docs/concepts/storage.md

```diff
@@ -0,0 +1,97 @@
+---
+id: storage
+title: Storage Staking
+sidebar_label: Storage Staking
+---
+
+If you’re familiar with Ethereum’s pricing model, you may know that the protocol charges a fee (called "gas") once for a transaction, and this transaction takes into account the amount of data stored via that transaction. This essentially means that anyone can pay once to store permanent data on-chain. This is a poor economic design for at least two reasons: 1. The people running the network (miners, in the case of Ethereum 1) are not appropriately incentivized to store large amounts of data, since a gas fee far charged in the distant past can increase storage costs forever, and 2. The users of a smart contract are charged for the data they send to store in it, rather than charging the owner of the smart contract.
+
+NEAR addresses this economic misalignment using a mechanism called storage staking (you may have heard that NEAR charges rent for storage – this has not been true [since March 2020](https://github.com/nearprotocol/nearcore/pull/2272)). In storage staking (sometimes called _state_ staking), the account that owns a smart contract must stake (or lock) tokens according to the amount of data stored in that smart contract, effectively reducing the balance of the contract's account.
```

Comment:
If we start with this paragraph, this doc will be much more friendly for a non-blockchain but technically solid reader.

Reply:
Great idea, thanks! Rearranged.